presto-main/src/main/java/com/facebook/presto/cost/VariableStatsEstimate.java

```diff
@@ -99,6 +115,15 @@ public double getNullsFraction()
         return nullsFraction;
     }
 
+    // We ignore the histogram during serialization because histograms can be
+    // quite large. Histograms are not used outside the coordinator, so there
+    // isn't a need to serialize them
+    @JsonIgnore
```

Comment:
@rschlussel This is the change to prevent serializing histograms. An associated test is added in [TestVariableStatsEstimate](https://github.com/prestodb/presto/blob/43ef3e7ba04324eb2a08d15d3d6bd9fb4be78e7f/presto-main/src/test/java/com/facebook/presto/cost/TestVariableStatsEstimate.java)